src/app/about/about.component.html

```diff
@@ -1 +1,12 @@
-<h1>about works!</h1>
+<div class="about-section">
+        <div class="inner-container">
+            <h1>About Us</h1>
+            <p class="text">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Doloribus velit ducimus, enim inventore earum, eligendi nostrum pariatur necessitatibus eius dicta a voluptates sit deleniti autem error eos totam nisi neque voluptates sit deleniti autem error eos totam nisi neque.
+            </p>
+            <div class="field">
+                <span>Heart condition prediction </span>
+                <span>Patient data management</span>
+            </div>
+        </div>
+    </div>
```

Comment:
Put a new line here at end of file this could help prevent the errors caused by TSLint :)